go.mod

```diff
@@ -223,23 +223,26 @@ require (
 	gopkg.in/op/go-logging.v1 v1.0.0-20160211212156-b2cb9fa56473 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/cli-runtime v0.21.0 // indirect
-	k8s.io/gengo v0.0.0-20201214224949-b6c5ce23f027 // indirect
-	k8s.io/klog/v2 v2.9.0 // indirect
-	k8s.io/kubectl v0.21.0 // indirect
+	k8s.io/cli-runtime v0.23.1 // indirect
+	k8s.io/gengo v0.0.0-20210813121822-485abfe95c7c // indirect
+	k8s.io/klog/v2 v2.30.0 // indirect
+	k8s.io/kubectl v0.23.1 // indirect
 	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.22 // indirect
-	sigs.k8s.io/kustomize/api v0.8.5 // indirect
-	sigs.k8s.io/kustomize/kyaml v0.10.15 // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
-	sigs.k8s.io/yaml v1.2.0 // indirect
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.25 // indirect
+	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
+	sigs.k8s.io/kustomize/api v0.10.1 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.13.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.0 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
 	// controller runtime
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20211014063134-be2a7fb8aa44
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 
+	github.com/operator-framework/api v0.11.1 => github.com/operator-framework/api v0.11.2-0.20220118211235-330a17e17dca
```

Comment:
I think my preference would be to avoid coupling the 1.23 kube dependency bump with an o-f/api bump as well (e.g. those changes are handled in it's own PR, or as an additional commit in your optional manifest work). It would help reduce the scope of this PR, and cut down on the overall diff. Any thoughts? In any case, I think we still need to properly cut an o-f/api tag for those CRD changes you had merged.

Reply:
I am not sure that what you are proposing is feasible. The kube 1.23 bump was driven by  the need to bump o-f/api. The latest o-f/api version comes with kube 1.23 dependencies and a newer logging stack.

Reply:
Yeah I should've been clearer: I think we can bump the kube 1.21 dependencies + the logr dependency manually, without having to bump o-f/api at the same time. I'm still working through the set of required changes, and viewing the overall diff is fairly disruptive, so maybe it's easier to simply chunk out some of the overall set of changes into their own commits (without having to create another PR), or we can simply bite the bullet for now if this is urgent enough.